### PR TITLE
Starting doc for new admins

### PR DESCRIPTION
**Pull request by KellerFuchs**
Do not merge yet, I have more things to say

### CODE REVIEW

**Comment by RyanSquared** [comment]
Is it possible to do something like `echo "$(cat | head -n1)"` for the passwords?

**Comment by KellerFuchs** [comment]
@ChickenNuggers It is, but it means the passwords have to be copy-pasted more times, and it doesn't protect about pasting (by right click or otherwise) at the wrong moment.

I have no strong opinion on that issue, though.

**Comment by daurnimator** [comment]
Formatting nitpick: when writing markdown, have a single empty newline after a heading; and 2 empty lines at the end of a paragraph/section.

**Comment by RyanSquared** [comment]
Can we also add in how to reset the shell keys for a user? I have no idea how, and having that added would be a help.

**Comment by KellerFuchs** [comment]
@ChickenNuggers Definitely worth adding, probably not in `New_admin.md`

**Comment by RyanSquared** [comment]
Alright. I suppose we should make a new document relating to recovering services and access, then?

**Comment by RyanSquared** [comment]
> Alright. I suppose we should make a new document relating to recovering services and access, then?

Which would also be useful for adding/changing shell keys to the DigitalOcean servers.

**Comment by RyanSquared** [comment]
Also, since I guess this may be kinda relevant without opening a new issue, we should implement some form of account recovery through GitHub while we still use it..

**Comment by KellerFuchs** [comment]
You mean some kind of automated account recovery,
  or some agreed-upon procedure for checking that
  a given user owns the account?

Also, wrong issue for this, I think.

**Comment by RyanSquared** [comment]
>   some agreed-upon procedure for checking that
>   a given user owns the account?

Correct

File: doc/New_admin.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+		        password "$HASH" { sha1; };
+		        flags { netadmin; can_gkline; can_override; can_gzline; can_restart; can_die; global; };
+		};
+
+5. `docker exec -it ircd Unreal3.2.10.4/unreal rehash`
+
+Now, in your IRC client, you can `/oper $USER $PASSWORD`.
+
+_TODO:_ The IRC config ought to be an Ansible-managed template.
+However, I don't think we should store password hashes in plaintext here.
```
Comment (drGrove) [inline doc/New_admin.md:96]: @KellerFuchs what are your thoughts for this? Do you know of a good solution?

Comment (KellerFuchs) [inline doc/New_admin.md:96]: @drGrove  The solution isn't very hard, it just `ENOTIME`d very hard lately.
Basically, it's possible/easy to write a jinja2 template and tell Ansible to instantiate the template as `unrealircd/opers.conf`, and the template can easily loop over a list of users.

**Comment by KellerFuchs** [comment]
Ok, I created separate issues for everything that didn't belong here, and handled the two relevent comments (thanks @AlissaSquared and @daurnimator).

**Pull request merged**